packages/fetchai/connections/p2p_noise/aea/api.go

```diff
@@ -78,14 +78,14 @@ func (aea *AeaApi) Init() error {
   uri := os.Getenv("URI")
 
   if aea.msgin_path == "" || aea.msgout_path == "" || aea.id == "" || uri == "" {
-    fmt.Println("[aea-api][error] couldn't get configuration")
+    fmt.Println("[aea-api  ][error] couldn't get configuration")
```

Comment:
don't understand why all the whitespace is added

Reply:
for `aea-api` logs to align with `noise-p2p`, a hack in the absence of a proper logger.
will be fixed in a subsequent pr